test/unit/data.js

```diff
@@ -773,7 +773,12 @@ test("jQuery.acceptData", 11, function() {
 	ok( jQuery.acceptData( document.documentElement ), "documentElement" );
 	ok( jQuery.acceptData( {} ), "object" );
 	ok( jQuery.acceptData( document.createElement( "embed" ) ), "embed" );
-	ok( jQuery.acceptData( document.createElement( "applet" ) ), "applet" );
+
+	// BrowserStack regularly has outdated Java causing IE to hang on this line.
+	// Skip it on TestSwarm.
+	if ( location.search.indexOf( "swarmURL=" ) === -1 ) {
+		ok( jQuery.acceptData( document.createElement( "applet" ) ), "applet" );
+	}
```

Comment:
Don't do this; just drop the assertion unconditionally and let the preceding one cover it. See https://github.com/jquery/jquery/issues/1938#issuecomment-67330082 ("we don't need an **`applet`** or applet `object`") and observe that applet and embed get identical treatment: https://github.com/jquery/jquery/blob/7b111310977a30d02a8260a7ed6c9c437bb1b0a5/src/data.js#L242-L249